src/header/js/header-menus.js

```diff
@@ -237,6 +244,68 @@ export default ({variant = 'desktop'}) => {
 		);
 	});
 
+	function resetFlyoutDimensions(parentNode) {
+		const subMenus = document.querySelectorAll('.esri-header-menus-submenu');
+		const parent = (parentNode !== 'disabled' && parentNode !== 'init') && parentNode.getAttribute('data-parent');
+		const parentElement = document.querySelector(`#${parent}`);
+
+		if (parentNode === 'init') {
```

Comment:
Is ParentNode a node or a string?  The name makes me think it should be a DOM element, but I see it's just a string in the if statments